### DISCUSSION TITLE
Add "noopener" or "noreferrer" to external links

### DISCUSSION THREAD

**Discussion by epinzur** [quarto-dev/quarto-cli#7137] [Feature Requests]
### Description

It would be great if quarto automatically updated external links to include `rel="noopener"` or `rel="noreferrer"`

See https://developer.chrome.com/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/

**Comment by mcanouil** [comment]
@dragonstyle I believe this would be easy to add alongside the external links feature (https://quarto.org/docs/output-formats/html-basics.html#external-links).
What do you think?

**Reply by dragonstyle** [reply]
I think this would be easy (and I enabled it for our next pre-release build)!

f0cd90d58e6a5b327a2631844742a5da946e9710

**Reply by mcanouil** [reply]
Thanks @dragonstyle!
Closing this as resolved!
